README.md

```diff
@@ -47,6 +47,27 @@ Go Report Card is an open source project run by volunteers, and contributions ar
 
 The code is licensed under the permissive Apache v2.0 licence. This means you can do what you like with the software, as long as you include the required notices. [Read this](https://tldrlegal.com/license/apache-license-2.0-(apache-2.0)) for a summary.
 
-### Notes
+### MS Windows
+
+Our users reported: Go Report Card works very well on Windows.
+You just run it is as usual (no need for the scripts).
+
+- You ***must*** have [`gometalinter`]() installed and available in Your `%path%`
```

Comment:
Can we shorten this a bit please, e.g. 

-----------

Go Report Card does not officially support Windows, but users report that it works. Instead of the installation steps detailed above, install gometalinter directly:

```
go get github.com/alecthomas/gometalinter
go install github.com/alecthomas/gometalinter
```

You will also need the `wc` word count unix tool available in your path.

Reply:
Even before I went to sleep, I separated the windows stuff into a separate dedicated page, and leave the main `ReadMe.md` short and sweet.
(just, I didn't commit, as I wanted to sleep about it.)

Reply:
`wc` is optional, is it not?
(I happen to have it anyway - I'm more unix than it may seem)
But Your error handling looks very graceful to me :-)